src/main/java/pl/confitura/jelatyna/admin/Controller.java

```diff
@@ -0,0 +1,54 @@
+package pl.confitura.jelatyna.admin;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.confitura.jelatyna.common.TokenGenerator;
+import pl.confitura.jelatyna.email.EmailService;
+
+import javax.validation.Valid;
+import java.util.List;
+
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+import static org.springframework.web.bind.annotation.RequestMethod.PUT;
+
+@RestController("adminController")
+@RequestMapping("/api/admin")
+public class Controller {
```

Comment:
1. Zgadzam się, to istotne. więc może warto sprobować zmienić przyzwyczajenie i zamiast `AdminController` napisać `admin.Controller` albo `a.Cont` albo jakakolwiek wariacja. Tutaj jedyną różnicą jest przyzwyczajenie

2. spoko, ten pull request nie jest jeszcze skonczony. Chciałem wrzucić żeby właśnie przedyskutować nazewnictwo Controllerow i innych obiektów.